packages/bpk-component-fieldset/src/BpkFieldset.js

```diff
@@ -60,9 +60,19 @@ const BpkFieldset = (props: Props) => {
     return null;
   }
 
+  let childId = children.props.id;
+  if (
+    children.props.inputProps &&
```

Comment:
This is redundant `children.props.inputProps && children.props.inputProps.id` is enough